src/main/java/ru/yandex/practicum/filmorate/service/FilmService.java

```diff
@@ -0,0 +1,77 @@
+package ru.yandex.practicum.filmorate.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ru.yandex.practicum.filmorate.exception.NotFoundException;
+import ru.yandex.practicum.filmorate.exception.ValidationException;
+import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.model.User;
+import ru.yandex.practicum.filmorate.storage.film.FilmStorage;
+import ru.yandex.practicum.filmorate.storage.user.UserStorage;
+
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+
+
+@Service
+public class FilmService {
+    private final FilmStorage filmStorage;
+    private final UserStorage userStorage;
+
+    @Autowired
+    public FilmService(FilmStorage filmStorage, UserStorage userStorage) {
+        this.filmStorage = filmStorage;
+        this.userStorage = userStorage;
+    }
+
+    public Film create(Film film) {
+        return filmStorage.create(film);
+    }
+
+    public Film update(Film film) {
+        return filmStorage.update(film);
+    }
+
+    public Film getById(int id) {
+        return filmStorage.getById(id);
+    }
+
+    public Collection<Film> getAll() {
+        return filmStorage.getAll();
+    }
+
+    public void addLike(int filmId, int userId) {
+        Film film = filmStorage.getById(filmId);
+        User user = userStorage.getById(userId);
+
+        if (film.getLikes().contains(userId)) {
+            throw new ValidationException("Пользователь уже поставил лайк");
+        }
+        film.getLikes().add(userId);
+    }
+
+    public void removeLike(int filmId, int userId) {
+        Film film = filmStorage.getById(filmId);
+        userStorage.getById(userId);
+
+        if (!film.getLikes().contains(userId)) {
+            throw new NotFoundException("Лайк не найден");
+        }
+        film.getLikes().remove(userId);
+    }
+
+    public List<Film> getPopularFilms(int count) {
+        List<Film> allFilms = filmStorage.getAll().stream()
```

Comment:
```suggestion
        return filmStorage.getAll().stream()
```
Здесь переменная используется только для того, чтобы вернуть значение метода. Лучше просто сразу писать `return` и вызывать нужный метод